packaging/language/easy_install.py

```diff
@@ -89,8 +89,9 @@
 - easy_install: name=bottle virtualenv=/webapps/myapp/venv
 '''
 
-def _is_package_installed(module, name, easy_install):
-    cmd = '%s --dry-run %s' % (easy_install, name)
+def _is_package_installed(module, name, easy_install, executable_arguments):
+    executable_arguments.append('--dry-run')
```

Comment:
Won't this mean that all further uses of executable_arguments outside of the function will contain '--dry-run'?